pkg/reconciler/taskrun/entrypoint/entrypoint.go

```diff
@@ -41,6 +41,8 @@ const (
 	DownwardMountName      = "downward"
 	DownwardMountPoint     = "/builder/downward"
 	DownwardMountReadyFile = "ready"
+	LogCfgMount            = "log-config"
```

Comment:
This code has all moved to pkg/pod, but it looks like this change should be just as easy to make there.